File: db4o.net/Db4objects.Db4o.Tests/native/Db4objects.Db4o.Tests/CLI2/Collections/Transparent/CollectionHolder.cs
/* Copyright (C) 2010  Versant Inc.   http://www.db4o.com */

using System;

namespace Db4objects.Db4o.Tests.CLI2.Collections.Transparent
{
	[Serializable]
	public class CollectionHolder<TColl>
	{
		public CollectionHolder(TColl collection)
		{
			_collection = collection;	
		}

		public TColl Collection
		{
			get { return _collection; }
		}

#if SILVERLIGHT
		public TColl _collection;
#else
		private readonly TColl _collection;
#endif
	}
}

File: db4oj.tests/src/com/db4o/db4ounit/jre12/collections/transparent/CollectionHolder.java
/* Copyright (C) 2009  Versant Inc.   http://www.db4o.com */

package com.db4o.db4ounit.jre12.collections.transparent;

import java.util.*;

/**
 * @sharpen.remove
 */
@decaf.Remove(decaf.Platform.JDK11)
public class CollectionHolder <C extends Collection<CollectionElement>>{
	public C _collection;
}

